lib/components/Nav.jsx

```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
     <nav
       className='justify-end items-center hidden sm:flex w-2/3'
     >
+      <Link
```

Comment:
The next.js Link component isn't necessary here as that helps with the SPA routing. We can simply use a regular `<a>` tag with the `href` attribute.